src/main/webapp/app/shared/link-preview/components/link-preview-container/link-preview-container.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { LinkPreview, LinkPreviewService } from 'app/shared/link-preview/services/link-preview.service';
+import { Link, LinkifyService } from 'app/shared/link-preview/services/linkify.service';
+
+@Component({
+    selector: 'jhi-link-preview-container',
+    templateUrl: './link-preview-container.component.html',
+    styleUrls: ['./link-preview-container.component.scss'],
+})
+export class LinkPreviewContainerComponent implements OnInit {
+    // to forward
+    @Input() color = 'primary'; // accent | warn
+    @Input() multiple: boolean;
+    @Input() data: string | undefined;
+
+    linkPreviews: LinkPreview[] = [];
+    hasError: boolean;
+    loaded = false;
+    showLoadingsProgress = true;
+
+    constructor(public linkPreviewService: LinkPreviewService, public linkifyService: LinkifyService) {}
+
+    ngOnInit() {
+        this.data = this.data ?? '';
+        const links: Link[] = this.linkifyService.find(this.data!);
+        // TODO: The limit of 5 link previews should be configurable (maybe in course level)
```

Comment:
Why should instructors set this limit? I don't see a reason why one course would use more/less previews than another

Reply:
We had the idea at the beginning to make this configurable but also tools like slack limit to fixed number (5 preview per message) so maybe we can even leave as it is. 

I wanted to have a reminder for what we thought with @krusche